azurerm/internal/services/netapp/tests/resource_arm_netapp_snapshot_test.go

```diff
@@ -204,6 +219,30 @@ resource "azurerm_netapp_snapshot" "test" {
   volume_name         = azurerm_netapp_volume.test.name
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
+
+  tags = {
+    "foo" = "bar"
+  }
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMNetAppSnapshot_updateTags(data acceptance.TestData) string {
+	template := testAccAzureRMNetAppSnapshot_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_netapp_snapshot" "test" {
+  name                = "acctest-NetAppSnapshot-%d"
+  account_name        = azurerm_netapp_account.test.name
+  pool_name           = azurerm_netapp_pool.test.name
+  volume_name         = azurerm_netapp_volume.test.name
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+
+  tags = {
+    "foo" = "baz"
```

Comment:
Could we ensure capitals work
```suggestion
    "FoO" = "BaR"
```